tests/test_actions.py

```diff
@@ -111,3 +119,78 @@ def test_stop_commands(self, capsys):
     def test_resume_command(self, capsys):
         actions.command("pause-logging")
         assert self._stdout(capsys) == "::pause-logging::"
+
+
+class TestSetenv:
+
+    """Tests for gidgethub.actions.setenv()."""
+
+    def test_creating(self, tmp_envfile):
+        actions.setenv("HELLO", "WORLD")
+        data = tmp_envfile.read_text(encoding="utf-8")
+        assert os.environ["HELLO"] == "WORLD"
+        assert (
+            data == f"HELLO<<__GIDGETHUB_DELIMITER__{os.linesep}WORLD{os.linesep}"
+            f"__GIDGETHUB_DELIMITER__{os.linesep}"
```

Comment:
Should we use a constant instead of hard coding the delimiter value in here and other tests?

Reply:
Yeah, I think it's fine to grab it from the module directly to avoid any future name changes breaking the tests.